portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase

```diff
@@ -357,10 +357,8 @@ definition {
 			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
 
 		FormFields.selectSelectOption(
-			fieldName = "If Input",
-			selectOption = "Is not email");
-
-		FormFields.viewTextFieldNotPresent(fieldLabel = "The Value");
```

Comment:
@gleicelisbino, shouldn't we keep this step?

Reply:
I think not. because, the test validation occurs twice. For example, when is a email, an Assert occurs and the same for when not a email. If dont possible clicked in the option, the test fail.  The third test don't use this. Other problem is, don't exist in the Form, "The Value", just "Value". As, this input it's a simple word, do match with "Predenifed Value" in the path. Then, I can not use ElementNotPresent because "Value" is present. I can not use "=" in the path, because the word just is mapped with "contains", and the assert ocorrs in the first word. What do you think? @natocesarrego